.github/workflows/run-notebook-tests-on-last-released-version.yml

```diff
@@ -0,0 +1,44 @@
+name: Notebook-Tests--Against-Last-Release
```

Comment:
What's the alerting around this, if the build starts failing?

Reply:
Correct. You can still complete it but it'll make sure whoever completes it knows that it's not compatible anymore.

Reply:
Like in this PR for example!

Reply:
so if I rerun this now it should hopefully succeed... due to the release

Reply:
It failed, perhaps needs latest master to update the notebooks first. Rerunning after git pull origin master

Reply:
Oh this changed already since the release. Miro's metrics changes are required for two notebooks. So the failure is actually correct!